test/World_Test.java

```diff
@@ -0,0 +1,209 @@
+     * No Iterations
+     */
+    /*
+    -   Test to Make Sure Percent Alive of 65 Works
+    -   Uses Default Values for Size and Seed
+    -   Expected Results:   World.toString() results matches World.toString_OLD() results
+     */
+    @Test
+    public void test_toString_CompareVersions_Alive_65()
+    {
+        World w = new World(SIZE, SEED, 65);
+        assertEquals(w.toString_OLD(), w.toString());
+    }
+
+    /*
+    -   Test to Make Sure Percent Alive of 0 Works -> Edge Case
+    -   Uses Default Values for Size and Seed
+    -   Expected Results:   World.toString() results matches World.toString_OLD() results
+     */
+    @Test
+    public void test_toString_CompareVersions_Alive_0()
+    {
+        World w = new World(SIZE, SEED, 0);
+        assertEquals(w.toString_OLD(), w.toString());
+    }
+
+    /*
+    -   Test to Make Sure Percent Alive of 100 Works -> Edge Case
+    -   Uses Default Values for Size and Seed
+    -   Expected Results:   World.toString() results matches World.toString_OLD() results
+     */
+    @Test
+    public void test_toString_CompareVersions_Alive_100()
+    {
+        World w = new World(SIZE, SEED, 100);
+        assertEquals(w.toString_OLD(), w.toString());
+    }
+
+
+    /**
+     * The Following Tests ensure that changing the Percent Alive Works
+     * Includes 1 Iteration
+     */
+    /*
+    -   Test to Make Sure Percent Alive of 65 Works
+    -   Uses Default Values for Size and Seed
+    -   Iterate Once to make sure Iterations does not effect Results of World.toString()
+    -   Expected Results:   World.toString() results matches World.toString_OLD() results
+     */
+    @Test
+    public void test_toString_CompareVersions_Alive_65_Iteration()
+    {
+        World w = new World(SIZE, SEED, 65);
+        w = w.iterate();
+        assertEquals(w.toString_OLD(), w.toString());
+    }
+
+    /*
+    -   Test to Make Sure Percent Alive of 0 Works -> Edge Case
+    -   Uses Default Values for Size and Seed
+    -   Iterate Once to make sure Iterations does not effect Results of World.toString()
+    -   Expected Results:   World.toString() results matches World.toString_OLD() results
+     */
+    @Test
+    public void test_toString_CompareVersions_Alive_0_Iteration()
+    {
+        World w = new World(SIZE, SEED, 0);
+        w = w.iterate();
+        assertEquals(w.toString_OLD(), w.toString());
+    }
+
+    /*
+    -   Test to Make Sure Percent Alive of 100 Works -> Edge Case
+    -   Uses Default Values for Size and Seed
+    -   Iterate Once to make sure Iterations does not effect Results of World.toString()
+    -   Expected Results:   World.toString() results matches World.toString_OLD() results
+     */
+    @Test
+    public void test_toString_CompareVersions_Alive_100_Iteration()
+    {
+        World w = new World(SIZE, SEED, 100);
+        w = w.iterate();
+        assertEquals(w.toString_OLD(), w.toString());
+    }
+}
```

Comment:
Great comments.  While I like the idea of comparing with the OLD method, should be comparing against explicit expected behavior.